### ISSUE TITLE
Add code field for paper BibTeX

### ISSUE THREAD

**Issue by johannesjmeyer** [quantum-journal/o3po#185]
In the quantum bibstyle, a new non-standard `code` field is introduced that allows authors to add code to references, e.g. GitHub repositories. It would be good if that feature could also be supported in O-3PO and if it would be rendered in the BibTeX.

Seen that Crossref creates its own bibliography exports from the Metadata, maybe this should also be added there somehow, maybe through a `note` tag, but I don't know how CrossRef works so I can't elaborate on the ways this could be properly exported there.

**Comment by cgogolin** [comment]
Is there a paper on the arXiv that uses this feature that I could use for development and testing?

**Comment by johannesjmeyer** [comment]
No, sadly not because that was our invention as far as I recall
